docs/docs/ecosystem/open-source-products.md

```diff
@@ -0,0 +1,82 @@
+---
+id: products
```

Comment:
Regarding IDs and file names, see my comment in the base review comment. This must match the file name, so if this was `projects.md` it needs to be:

```suggestion
id: projects
```